.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -5,38 +5,109 @@ body:
-        please be sure to file it there.
+        Before filing a bug, please be sure you have searched through [existing open bugs](https://github.com/open-telemetry/opentelemetry-js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug)
+        to see if this bug has already been filed.
+  - type: markdown
+    attributes:
+      value: |
+        ## Basic Troubleshooting
+  - type: checkboxes
+    id: updated-to-latest
+    attributes:
+      description: |
+        > [!NOTE]
+        > If you depend on experimental packages with a range like `opentelemetry/sdk-node@^0.53.0` that is equivalent
+        > to `@opentelemetry/sdk-node@0.53.x`, NOT `@opentelemetry/sdk-node@0.x.x`. `0.54.x` **WILL NOT** be picked up
+        > by this constraint due to the way semantic versioning works for `0.x` packages. This is intentional, see
+        > https://semver.org/ for details.
+      label: Updating packages
+      options:
+        - label: I have updated all `@opentelemetry/*` packages but the issue persists
+          required: true
+  - type: dropdown
+    id: esm
+    attributes:
+      description: |
+        **How to find this information:**
+          - check your `package.json`
+            - if you have `"type": "commonjs"` or no `"type"` then you're likely using CommonJS
+            - if you have `"type": "module"`, you are likely using ESM - please ensure you have read the [instructions for ESM users](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) before opening an issue.
+          - other indicators:
+            - using `.js` or `.cjs` (CommonJS)
+            - using `.mjs` (ESM - please ensure you have read the [instructions for ESM users](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) before opening an issue.)
+      label: Which module type are you using?
+      multiple: false
+      options:
+        - CommonJS
+        - ESM
+        - I don't know
+  - type: checkboxes
+    id: bundling
+    attributes:
+      description: |
+        Bundling removes `require` and `import` statements, which results in most `@opentelemetry/instrumentation-*` packages not working as they rely on the presence
+        of these statements to function.
+      label: Bundling and `@opentelemetry/instrumentation-*`packages
+      options:
+        - label: I understand that `@opentelemetry/instrumentation-*` packages are not intended to work in bundles apps (with a few exceptions).
+          required: true
+  - type: markdown
+    attributes:
+      value: |
+        ## Bug Reproduction
+        > [!IMPORTANT]
+        >  **If you use a third-party package that re-distributes OpenTelemetry, open the bug ticket with that third party unless you can provide steps to reproduce this with pure OpenTelemetry.**
+        >  Digging into third-party distributions of OpenTelemetry is not in scope for this project.
```

Comment:
This IMPORTANT section could perhaps move to the top section along with the "Please read all parts of this form carefully. ..." paragraph. Whether it is a 3rd-party distro isn't really scoped to a reproduction.

Reply:
Sounds good, I moved it to the top in https://github.com/open-telemetry/opentelemetry-js/pull/5086/commits/d7245428fff8acda4187c264627cd13569d042a1